server/server.go

```diff
@@ -50,15 +50,21 @@ func Start(ip string, port int, cert string, key string, debug bool, t *Tracker,
 			continue
 		}
 
-		ip, _, _ := net.SplitHostPort(conn.RemoteAddr().String())
+		ip := getIP(conn)
 
 		context, err := limit.Get(nil, ip)
 		if err != nil {
+			if debugMode {
+				fmt.Printf("[Listener] Unable to get connection limit: %s\n", err)
```

Comment:
Might be nice to get all these prefixes as constants, would prevent misspellings later. =)

Reply:
Yeah I would like to standardize the internal logic of those labels a bit as well. Will ask for review when I get it done. Thanks for the suggestions.